barretenberg/Earthfile

```diff
@@ -29,15 +29,17 @@ barretenberg-acir-tests-bb:
     # This ensures we test independent pk construction through real/garbage witness data paths.
     RUN FLOW=prove_then_verify ./run_acir_tests.sh
     # Construct and separately verify a UltraHonk proof for a single program
-    RUN FLOW=prove_then_verify_ultra_honk ./run_acir_tests.sh double_verify_nested_proof
+    RUN FLOW=prove_then_verify_ultra_honk ./run_acir_tests.sh sha256
     # Construct and separately verify a MegaHonk proof for all acir programs
     RUN FLOW=prove_then_verify_mega_honk ./run_acir_tests.sh
     # Construct and verify a UltraHonk proof for a single program
-    RUN FLOW=prove_and_verify_ultra_honk ./run_acir_tests.sh double_verify_nested_proof
+    RUN FLOW=prove_and_verify_ultra_honk ./run_acir_tests.sh pedersen_hash
```

Comment:
Any reason this one is pedersen and the other is sha256? 
We should add a TODO for bringing back a double_verify_nested_proof

Reply:
nah, I was just picking different ones to increase variability. We should be able to bring this back in other PR which adds the verify_honk_proof test program.